test/int_processing_eras.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+// this file contains logic for processingeras API
+// the HTTP request body is defined by dbs.ProcessingEras struct defined in dbs/processingeras.go
+// the HTTP response body is defined by dbs.ProcessingEra struct defined in dbs/processingeras.go
```

Comment:
there is no `dbs.ProcessingEra` in current codebase, may be you added it in your fork and forget to add to this commit.